### PR TITLE
API Call + Code compiles and runs

### LINKED ISSUE #6: Compile code from the editor
Make a output window to run code.
`https://piston.readthedocs.io/en/latest/api-v2/`

### PR DESCRIPTION
**Pull request by yathansh13**
fixes #6

### CODE REVIEW

**Comment by DevanceJ** [comment]
good code, make issues on github for the things mentioned in comments
move `api.js` to the `lib` folder or create a `utils` folder

**Comment by DevanceJ** [comment]
`{
"language": "javascript",
"version": "1.32.3",
"aliases": [
"deno-js"
],
"runtime": "deno"
}`

change js version to this

**Comment by DevanceJ** [comment]
image
Errors are kind of unreadable, see if you can do something about this.

**Comment by yathansh13** [comment]
Fixes #1 
Also changed js version

File: frontend/src/components/code-editor.jsx [hunk 1]
```diff
@@ -20,13 +21,15 @@ const CodeEditor = () => {
       <LangSelect language={language} onSelect={onLangChange} />
       <Editor
         width="100%"
-        height="75vh"
+        height="55vh"
```
Comment (yathansh13) [inline frontend/src/components/code-editor.jsx:24]: Fix responsiveness

File: frontend/src/contants.js [hunk 2]
```diff
@@ -34,7 +34,17 @@ const BOILERPLATE = {
...
-  `
+  `,
 };
 
 export default BOILERPLATE;
+
+// GET /api/v2/runtimes <- modify to call from this endpoint for versions
+// Hardcoded for now to test functionality
+
+export const LANGUAGE_VERSIONS = {
```
Comment (yathansh13) [inline frontend/src/contants.js:45]: Get values from API

File: frontend/src/pages/Interview.jsx [hunk 3]
```diff
@@ -1,9 +1,36 @@
...
   return (
-    <div className=" m-4">
-      <CodeEditor />
+    <div className="flex items-center h-screen gap-[10px]">
+      <div className="flex justify-center items-center mt-2.5 w-9"></div>
+      <div className=" m-4 w-3/5">
+        <CodeEditor />
+      </div>
+      <div className="flex flex-col gap-[54px]">
+        <div className="w-[450px]">
```
Comment (yathansh13) [inline frontend/src/pages/Interview.jsx:13]: Check when making responsive

**Pull request merged**